recipes/example/meta-compiled.yaml

```diff
@@ -0,0 +1,103 @@
+# This recipe example is for packages which are compiled libraries and binaries.
```

Comment:
Would a more realistic example be a python package that uses compiled binaries/libraries? That example may prove more useful to explain the linking and prefix location, etc.